contracts/CTKNCrowdsale.sol

```diff
@@ -1,90 +1,31 @@
 pragma solidity ^0.4.19;
 
 import 'zeppelin-solidity/contracts/math/SafeMath.sol';
-import 'zeppelin-solidity/contracts/crowdsale/RefundableCrowdsale.sol';
-import 'zeppelin-solidity/contracts/crowdsale/CappedCrowdsale.sol';
-import 'zeppelin-solidity/contracts/crowdsale/RefundVault.sol';
+import 'zeppelin-solidity/contracts/crowdsale/distribution/RefundableCrowdsale.sol';
+import 'zeppelin-solidity/contracts/crowdsale/distribution/utils/RefundVault.sol';
+import 'zeppelin-solidity/contracts/crowdsale/validation/CappedCrowdsale.sol';
+import 'zeppelin-solidity/contracts/crowdsale/emission/MintedCrowdsale.sol';
 import 'zeppelin-solidity/contracts/token/ERC20/MintableToken.sol';
 
 
-contract CTKNCrowdsale is CappedCrowdsale, RefundableCrowdsale {
+contract CTKNCrowdsale is CappedCrowdsale, RefundableCrowdsale, MintedCrowdsale {
     using SafeMath for uint256;
 
-    RefundVault public refundWallet;
-
-    mapping(address => uint256) private refunds;
-    address[] private refundees;
-
-    function Crowdsale(
-        uint256 _cap,
-        uint256 _goal,
-        uint256 _startTime,
-        uint256 _endTime,
+    function CTKNCrowdsale(
+        uint256 _openingTime,
+        uint256 _closingTime,
         uint256 _rate,
```

Comment:
Just to clarify is this the rate at which we will buy/convert eth to usd for the $54 share price at end of crowdsale?  So if we say we'll buy eth @ $1080 USD and so 20 shares per eth per end of crowdsale?